back/sssscs/src/main/java/com/ib/user/User.java

```diff
@@ -69,4 +81,8 @@ public boolean isCredentialsNonExpired() {
 	public boolean isEnabled() {
 		return true;
 	}
+	
+	public boolean equals(User other) {
+		return getId().equals(other.getId());
+	}
```

Comment:
I think I added it there because the default `equal` checks against `Object`, and I wanted to be sure that we compare them by `Id`. I guess it's not necessary. [bec5d02](https://github.com/magley/sssscs/pull/3/commits/bec5d020aa5ec3f306ff5b6034091a30fadcff31)